src/main/java/appeng/core/api/definitions/ApiParts.java

```diff
@@ -55,6 +55,7 @@
 	private final IItemDefinition importBus;
 	private final IItemDefinition exportBus;
 	private final IItemDefinition iface;
+	private final IItemDefinition fluidIface;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "fluidIface" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1068)